buildbot

```diff
@@ -12,9 +12,13 @@ def echo_run(cmd):
     print('+', *cmd)
     run(cmd, check=True)
 
+def get_vm_name():
+    process = run(['kitchen', 'list', '--bare'], stdout=PIPE)
```

Comment:
Maybe we want to check the return code of this command?
Also, how does this work if `kitchen list` lists two vms?